e2e-test/civiform_config_aws_oidc.sh

```diff
@@ -231,6 +231,11 @@ export APPLICANT_OIDC_PROVIDER_NAME="OidcClient"
 # Usually ends in .well-known/openid-configuration
 export APPLICANT_OIDC_DISCOVERY_URI="https://civiform-staging.us.auth0.com/.well-known/openid-configuration"
 
+# REQUIRED if CIVIFORM_APPLICANT_IDP="generic-oidc"
+# The URL applicants are redirected to for creating an account
+# with the identity provider.
+export APPLICANT_REGISTER_URI=""
```

Comment:
**suggestion (blocking):** can this be set to the auth0 account registration URL?

Reply:
Do you know where I would find that? I looked in our auth0 account, but we only have urls set for login and logout.

Reply:
Looks like it's the same as the login URL but with `screen_hint=signup` added to the params

https://community.auth0.com/t/how-do-i-redirect-users-directly-to-the-hosted-signup-page/42520

Reply:
chatted with bion about this and we're going to add this value in staging so it's testable.